src/libServerV2/Server.cpp

```diff
@@ -0,0 +1,1216 @@
+/*
```

Comment:
All the type, uint256_t should be protobuffed during response and request.

Reply:
Okay